python/tests/core/metrics/test_condition_count.py

```diff
@@ -0,0 +1,61 @@
+import re
+from typing import Dict
+
+from whylogs.core.dataset_profile import DatasetProfile
+from whylogs.core.datatypes import DataType
+from whylogs.core.metrics import Metric
+from whylogs.core.metrics.condition_count_metric import (
+    ConditionCountConfig,
+    ConditionCountMetric,
+)
+from whylogs.core.preprocessing import PreprocessedColumn
+from whylogs.core.resolvers import Resolver
+from whylogs.core.schema import ColumnSchema, DatasetSchema
+
+
+def test_condition_count_metric() -> None:
+    conditions = {
+        "alpha": re.compile("[a-zA-Z]+"),
+        "digit": re.compile("[0-9]+"),
+    }
```

Comment:
The plan is to add support for additional predicates once we're happy with the plumbing. It would be nice to do the `compile` behind the scenes.